backend/src/main/java/com/soen341/instagram/controller/PictureController.java

```diff
@@ -0,0 +1,53 @@
+package com.soen341.instagram.controller;
+
+import com.soen341.instagram.dao.impl.AccountRepository;
+import com.soen341.instagram.dto.picture.PictureDTO;
+import com.soen341.instagram.model.Account;
+import com.soen341.instagram.model.Picture;
+import com.soen341.instagram.service.impl.PictureService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+
+@RestController
+public class PictureController {
+
+    @Autowired
+    private AccountRepository accountRepository;
```

Comment:
We should avoid having a DOA object in the controller. This could be a later refactoring task.